src/ducks/SANCharts/ChartPage.js

```diff
@@ -526,24 +529,50 @@ class ChartPage extends Component {
               metrics.some(({ key }) => key === metricAnomalyKey)
             )
 
+          const metricsTool = (
+            <LoadableChartMetricsTool
+              classes={styles}
+              slug={slug}
+              toggleMetric={this.toggleMetric}
+              disabledMetrics={errorMetrics}
+              activeMetrics={finalMetrics}
+              activeEvents={events}
+              showToggleAnomalies={showToggleAnomalies}
+              onToggleAnomalies={this.onToggleAnomalies}
+              isShowAnomalies={isShowAnomalies}
+              alwaysShowingMetrics={alwaysShowingMetrics}
+              hideSettings={hideSettings}
+              isWideChart={isWideChart}
+            />
+          )
+
           return (
             <>
               {viewOnly || hideSettings.header || (
-                <Header
-                  slug={slug}
-                  isLoading={isParentLoading}
-                  isLoggedIn={isLoggedIn}
-                  onSlugSelect={this.onSlugSelect}
-                />
+                <>
+                  {isWideChart && <StoriesList classes={styles} />}
+                  <Header
+                    slug={slug}
+                    isLoading={isParentLoading}
+                    isLoggedIn={isLoggedIn}
+                    onSlugSelect={this.onSlugSelect}
+                  />
+                </>
               )}
               <div className={styles.wrapper}>
                 <div
                   className={cx(
                     styles.tool,
-                    isAdvancedView && styles.tool_short
+                    isAdvancedView && styles.tool_short,
+                    isWideChart && styles.tool_wide_chart
                   )}
                 >
-                  <div className={styles.container}>
+                  <div
+                    className={cx(
+                      styles.container,
+                      isWideChart && styles.wideChartBg
```

Comment:
suggest to use one naming convention - camelCase. So should be `toolShort` and `toolWideChart`
and use rules without nesting in scss